cli/src/main/java/org/jboss/as/cli/impl/CommandContextConfiguration.java

```diff
@@ -145,6 +145,10 @@ public Builder setClientBindAddress(String clientBindAddress) {
 
         public Builder setConsoleInput(InputStream consoleInput) {
             this.consoleInput = consoleInput;
+            if(consoleInput != null) {
```

Comment:
Should this only happen if this.errorOnInteract == null? So don't override whatever was previously set up via setErrorOnInteract().